index.html

```diff
@@ -22680,8 +22677,8 @@ <h3 id="examples_of_jukugoruby_distribution_with_intercharacter_space_expansion"
         </figcaption>
 
     </figure>
-    <p id="en-app6_5">[[[#fig_a6-12]]] are examples with the same jukugo-ruby as in [[[#fig_a6-10]]] and [[[#fig_a6-11]]] except their positions in the lines are at the line-head or at the line-end, where the base character and the ruby character at the line-head or at the line-end are both aligned.</p>
-    <p id="ja-app6_5"><span class="index" id="d18e796">[[[#fig_a6-12]]]の例</span>は，[[[#fig_a6-10]]]及び[[[#fig_a6-11]]]の例を行頭又は行末に配置し，親文字とルビ文字の先頭又は末尾をそろえる配置方法にした例である．</p>
+    <p ts-locale-filter-list="en" lang="en">[[[#fig_a6-12]]] are examples with the same jukugo-ruby as in [[[#fig_a6-10]]] and [[[#fig_a6-11]]] except their positions in the lines are at the line-head or at the line-end, where the base character and the ruby character at the line-head or at the line-end are both aligned.</p>
```

Comment:
s/ts/its/

Reply:
thank you for catching this!

Reply:
why are you removing the id attribute in this and the next change? 

and why is there extra markup and id (`<span class="index" id="d18e796">`) in the japanese that's not in the english?

Reply:
I'm not sure why only this item has id in 'xx-appX_X' (only exceptions are in list in section 3.9.2), and just removed to match with others. I can revert and re-add these two IDs.
On id for span added to class="index", JLReq TF discussed but no reason was found why it is added and concluded these are added during document processing for older editions (conversion to html format), so we just left them as is without actively removing nor adding.